specification/maps/data-plane/Render/preview/2.1/render.json

```diff
@@ -0,0 +1,451 @@
+      "required": true,
+      "type": "integer",
+      "format": "int32",
+      "x-ms-parameter-location": "method"
+    },
+    "Text": {
+      "name": "text",
+      "in": "query",
+      "description": "Yes/no value to exclude textual data from response. Only images and country names will be in response.",
+      "required": false,
+      "type": "string",
+      "enum": [
+        "yes",
+        "no"
+      ],
+      "default": "yes",
+      "x-ms-parameter-location": "method"
+    }
+  },
+  "paths": {
+    "/map/tile": {
+      "get": {
+        "description": "**Applies to**: S0 and S1 pricing tiers.\n\nThe Get Map Tiles API allows users to request map tiles in vector or raster formats typically to be integrated  into a map control or SDK. Some example tiles that can be requested are Azure Maps road tiles, real-time  Weather Radar tiles or the map tiles created using [Azure Maps Creator](https://aka.ms/amcreator). By default,  Azure Maps uses vector tiles for its web map control (Web SDK) and Android SDK.",
+        "operationId": "RenderV2_GetMapTilePreview",
+        "x-ms-examples": {
+          "GetMapTile": {
+            "$ref": "./examples/GetMapTileV2.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "../../../Common/preview/1.0/common.json#/parameters/ClientId"
+          },
+          {
+            "$ref": "#/parameters/RenderApiVersionV2"
+          },
+          {
+            "$ref": "#/parameters/TilesetId_rv2"
+          },
+          {
+            "$ref": "#/parameters/MapTileV2Zoom"
+          },
+          {
+            "$ref": "../1.0/render.json#/parameters/xTileIndex"
+          },
+          {
+            "$ref": "../1.0/render.json#/parameters/yTileIndex"
+          },
+          {
+            "name": "timeStamp",
+            "in": "query",
+            "description": "The desired date and time of the requested tile. This parameter must be specified in the standard date-time format (e.g. 2019-11-14T16:03:00-08:00), as defined by [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601). This parameter is only supported when tilesetId parameter is set to one of the values below.\n  \n* microsoft.weather.infrared.main: We provide tiles up to 3 hours in the past. Tiles are available in 10-minute intervals. We round the timeStamp value to the nearest 10-minute time frame.\n* microsoft.weather.radar.main: We provide tiles up to 1.5 hours in the past and up to 2 hours in the future. Tiles are available in 5-minute intervals. We round the timeStamp value to the nearest 5-minute time frame.",
+            "required": false,
+            "type": "string"
+          },
+          {
+            "name": "tileSize",
+            "in": "query",
+            "description": "The size of the returned map tile in pixels.",
+            "required": false,
+            "type": "string",
+            "default": "256",
```

Comment:
Should we let the backend decide the default tile size?

Reply:
hm... yeah, this shouldn't need a default, good catch